grafana/mattermost-notification-metrics.json

```diff
@@ -544,31 +521,44 @@
           {
             "matcher": {
               "id": "byName",
-              "options": "Total Notification Errors"
+              "options": "Total Notifications Unknown"
```

Comment:
What does `Unknown` mean in this context? It's not always an error in this category? Which part is this in the dashboard?

Reply:
The sent vs. not sent notifications for websocket:
<img width="264" alt="image" src="https://github.com/mattermost/mattermost-performance-assets/assets/52460000/8f66303d-8916-4d0b-81cb-ef3041db4f82">